k8s/policies.go

```diff
@@ -0,0 +1,238 @@
+package k8s
+
+import (
+	"reflect"
+	"strings"
+
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/yaml"
+	"k8s.io/client-go/kubernetes"
+	typedCorev1 "k8s.io/client-go/kubernetes/typed/core/v1"
+)
+
+const PolicyAnnotationKey = "com.openfaas.policies"
```

Comment:
I can see why you've gone with the plural, however do we use the plural for any other annotation, even where multiple values are supported? (perhaps all but `topic` are singular) The singular sounds more consistent.

Reply:
ok

Reply:
This was updated